pkgs/development/python-modules/python-watcherclient/default.nix

```diff
@@ -0,0 +1,76 @@
+{
+  lib,
+  buildPythonPackage,
+  cliff,
+  fetchFromGitea,
+  keystoneauth1,
+  openstackdocstheme,
+  osc-lib,
+  oslo-i18n,
+  oslo-serialization,
+  oslo-utils,
+  pbr,
+  pythonOlder,
+  setuptools,
+  sphinxcontrib-apidoc,
+  sphinxHook,
+  stestr,
+}:
+
+buildPythonPackage rec {
+  pname = "python-watcherclient";
+  version = "4.4.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitea {
```

Comment:
and same here

Reply:
Thank you for the review. I've handled all your feedbacks :)